README.md

```diff
@@ -0,0 +1,50 @@
+# **The Debuggers** 👩‍💻👨‍💻🕷
```

Comment:
can we change the name to cats coders ?😂

Reply:
it's even more suitable with the whole theme. 😭😭😭😭